api/api_full.go

```diff
@@ -63,6 +63,54 @@ type FullNode interface {
 	Common
 	Net
 
+	// MethodGroup: ChainIndexer
+	// The ChainIndexer method group contains methods for interacting with the chain indexer.
+
+	// IndexValidation contains detailed information about the validation status of a specific chain epoch.
+	//type IndexValidation struct {
```

Comment:
Since this is docs about IndexValidation, should these comments be moved to where IndexValidation is defined?